fairlearn/metrics/skm_wrappers.py

```diff
@@ -0,0 +1,95 @@
+        return skm.accuracy_score(y_true, y_pred,
+                                  normalize,
+                                  sample_weight)
+
+    return metric_by_group(internal_acc_wrapper, y_true, y_pred, group_membership, sample_weight)
+
+
+def group_confusion_matrix(y_true, y_pred, group_membership,
+                           labels=None,
+                           sample_weight=None):
+
+    def internal_cm_wrapper(y_true, y_pred, sample_weight=None):
+        return skm.confusion_matrix(y_true, y_pred,
+                                    labels,
+                                    sample_weight)
+
+    return metric_by_group(internal_cm_wrapper, y_true, y_pred, group_membership, sample_weight)
+
+
+def group_precision_score(y_true, y_pred, group_membership,
+                          labels=None, pos_label=1, average='binary',
+                          sample_weight=None):
+
+    def internal_prec_wrapper(y_true, y_pred, sample_weight=None):
+        return skm.precision_score(y_true, y_pred,
+                                   labels=labels, pos_label=pos_label, average=average,
+                                   sample_weight=sample_weight)
+
+    return metric_by_group(internal_prec_wrapper, y_true, y_pred, group_membership, sample_weight)
+
+
+def group_recall_score(y_true, y_pred, group_membership,
+                       labels=None, pos_label=1, average='binary',
+                       sample_weight=None):
+
+    def internal_recall_wrapper(y_true, y_pred, sample_weight=None):
+        return skm.recall_score(y_true, y_pred,
+                                labels=labels, pos_label=pos_label, average=average,
+                                sample_weight=sample_weight)
+
+    return metric_by_group(internal_recall_wrapper,
+                           y_true, y_pred, group_membership, sample_weight)
+
+
+def group_roc_auc_score(y_true, y_pred, group_membership,
+                        average='macro', max_fpr=None,
+                        sample_weight=None):
+
+    def internal_ras_wrapper(y_true, y_pred, sample_weight=None):
+        return skm.roc_auc_score(y_true, y_pred,
+                                 average=average, max_fpr=max_fpr,
+                                 sample_weight=sample_weight)
+
+    return metric_by_group(internal_ras_wrapper,
+                           y_true, y_pred, group_membership, sample_weight)
+
+
+def group_zero_one_loss(y_true, y_pred, group_membership,
+                        normalize=True,
+                        sample_weight=None):
+
+    def internal_zol_wrapper(y_true, y_pred, sample_weight=None):
+        return skm.zero_one_loss(y_true, y_pred,
+                                 normalize=normalize,
+                                 sample_weight=sample_weight)
+
+    return metric_by_group(internal_zol_wrapper, y_true, y_pred, group_membership, sample_weight)
+
+# --------------------------------------------------------------------------------------
```

Comment:
what's this for?

Reply:
The separator line?

Reply:
Yes :-) never seen this in python before. You can use new files if you want separation.

Reply:
In this case, they're all simple wrappers for `scikit-learn` and I'm just flagging where the regression ones begin.